doc/changelog.md

```diff
@@ -157,78 +293,72 @@
   ```
 
-* Add native support for `qml.PSWAP` and `qml.ISWAP` gates on Amazon Braket devices. Specifically, a circuit like
-
-  ```py
-  import pennylane as qml
-  from catalyst import qjit
+* Support has been added for linear algebra functions which depend on computing the eigenvalues
+  of symmetric matrices, such as `np.sqrt_matrix()`.
+  [(#488)](https://github.com/PennyLaneAI/catalyst/pull/488)
 
-  dev = qml.device("braket.local.qubit", wires=2, shots=100)
+  For example, you can compile `qml.math.sqrt_matrix`:
 
-  @qjit
-  @qml.qnode(dev)
-  def f(x: float):
-      qml.Hadamard(0)
-      qml.PSWAP(x, wires=[0, 1])
-      qml.ISWAP(wires=[1, 0])
-      return qml.probs()
+  ```python
+  @qml.qjit
+  def workflow(A):
+      B = qml.math.sqrt_matrix(A)
+      return B @ A
   ```
 
-  would no longer decompose the `PSWAP` and `ISWAP` gates to `SWAP`s, `CNOT`s and `Hadamard`s. Instead it would just call Braket's native `PSWAP` and `ISWAP` gates at runtime.
-  [(#458)](https://github.com/PennyLaneAI/catalyst/pull/458)
-
-* Add support for the `BlockEncode` operator in Catalyst.
-  [(#483)](https://github.com/PennyLaneAI/catalyst/pull/483)
+  Internally, this involves support for lowering the eigenvectors/values computation lapack method
+  `lapack_dsyevd` via `stablehlo.custom_call`.
 
-* Remove copies of TOML device configuration files for Lightning device in Catalyst.
-  [(#472)](https://github.com/PennyLaneAI/catalyst/pull/472)
+* Additional debugging functions are now available in the `catalyst.debug` directory.
+  [(#529)](https://github.com/PennyLaneAI/catalyst/pull/529)
+  [(#522)](https://github.com/PennyLaneAI/catalyst/pull/531)
 
-* Remove `qextract_p` and `qinst_p` from forced-order primitives.
-  [(#469)](https://github.com/PennyLaneAI/catalyst/pull/469)
+  This includes:
 
-* Update `AllocateQubit` and `AllocateQubits` in `LightningKokkosSimulator` to preserve
-  the current state-vector before qubit re-allocations.
-  [(#479)](https://github.com/PennyLaneAI/catalyst/pull/479)
+  - `compile_from_mlir(ir[, compiler, ...])` to compile a Catalyst function to binary code from the
+    provided MLIR.
 
-* Add support for post-selection in mid-circuit measurements. This is currently supported only on Lightning simulators.
-  [(#491)](https://github.com/PennyLaneAI/catalyst/pull/491)
+  - `filter_static_args(args, static_argnums)` to remove static values from arguments using the
+    provided index list.
 
-  This is an example of post-selection usage:
+  - `get_cmain(fn, *args)` to return a C program that calls a jitted function with the provided
+    arguments.
 
-  ```py
-  import pennylane as qml
-  from catalyst import qjit
+  - `print_compilation_stage(fn, stage)` to print one of the recorded compilation stages for a
+    JIT-compiled function.
 
-  dev = qml.device("lightning.qubit", wires=1)
+  - `promote_arguments(target_signature, args)` to promote arguments to the provided target
+    signature, preserving PyTrees.
```

Comment:
... and this are not debug functions. (they have been added to the new tracing module, and internal change)